projects/robots/pal_robotics/tiago++/worlds/tiago++.wbt

```diff
@@ -387,12 +383,12 @@ OilBarrel {
 }
 OilBarrel {
   translation 5.826 0.305 0.356
-  rotation -0.678 0.52 -0.52 -1.95
+  rotation -0.6778359835355757 0.5198742056615036 -0.5198742056615036 -1.95
```

Comment:
That is what Webots produces if you open and save the world.
If we don't want them, then we have to fix Webots.

Reply:
I understand this is the result of the normalization of the unit vector of the rotation, but it doesn't look very nice... 😒

Reply:
Any world saved with Webots will look like this because the rotation have to be a unit vector.
I agree that could be better to have a lower resolution, but until Webots is not fixed this is how the Webots files look like.

Reply:
OK, I understand.

Reply:
@omichel Do you want to complete your review?

Reply:
Unfortunately, I don't have time. Please, go ahead with the review.